Microsoft.Toolkit.Uwp.Services/Services/Bing/BingSearchConfig.cs

```diff
@@ -28,5 +28,10 @@ public class BingSearchConfig
         /// Gets or sets search query.
         /// </summary>
         public string Query { get; set; }
+
+        /// <summary>
+        /// Gets or sets search query news.
+        /// </summary>
+        public bool News { get; set; }
```

Comment:
Can we come up with a better name that better conveys that this switches to search news? `News` doesn't really follow the naming patterns for boolean properties.